sotodlib/core/metadata/obsdb.py

```diff
@@ -65,6 +80,11 @@ def __init__(self, map_file=None, init_db=True):
             if map_file is None:
                 map_file = ':memory:'
             self.conn = sqlite3.connect(map_file)
+        
+        global TABLE_DEFS
+        global TABLE_DEFS_OWB
+        if obs_wafer_band:
+            TABLE_DEFS = TABLE_DEFS_OWB
```

Comment:
This will modify the module level global var, affecting any future instantiations of ObsDb.  So instead you'll want to store the table_defs as an instance variable, and have all the internal functions refer to that. (`self._table_defs = TABLE_DEFS` / `TABLE_DEFS_OWB`).